src/gui/src/components/config/WordListForm.vue

```diff
@@ -130,7 +130,7 @@ export default {
       description: '',
       link: '',
       usage: [],
-      entries: []
```

Comment:
Maybe not my call to make, but I feel like, it is confusing to pass `entries` in create and update actions, but at the same time, not use them in the backend at all during create or update. 

I would like to suggest removing this key from being processed.

Reply:
This of course would "reflect" along the flow of the data mapping later on. So according adjustments would be needed.